# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/crashtests.list
# Commit: 83b018946c0b
# Full Hash: 83b018946c0b78a2035e6d50c3ccd55eac3570a4
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-06-01 04:09:30
# Description:
#   Bug 1771556 - Handle backdrop capture failing clip check when sub-graph is visible r=gfx-reviewers,lsalzman
#   
#   This can happen when there is a long nested chain of backdrop-filters
#   and float inaccuracies cause the final capture primitive to be culled.
#   
# ==============================================================================

diff -r d2a56d51c65b -r 83b018946c0b gfx/tests/crashtests/crashtests.list
--- a/gfx/tests/crashtests/crashtests.list	Tue May 31 22:21:15 2022 +0000
+++ b/gfx/tests/crashtests/crashtests.list	Tue May 31 22:55:32 2022 +0000
@@ -216,3 +216,4 @@
 load 1768096-1.html
 pref(layout.css.backdrop-filter.enabled,true) load 1771294.html
 pref(layout.css.backdrop-filter.enabled,true) load 1771293.html
+pref(layout.css.backdrop-filter.enabled,true) load 1771556.html